ts/input/tex/unicode/UnicodeConfiguration.ts

```diff
@@ -87,6 +90,7 @@ const UnicodeMethods: {[key: string]: ParseMethod} = {
         }
       }
     } else if (variant) {
+      console.log(12);
```

Comment:
The `console.log()` calls in this file need to be removed.

Reply:
Removed.